terra-batch-libchart/values.yaml

```diff
@@ -0,0 +1,51 @@
+exports:
+  common:
+    cromwell:
+      name: coa-cromwell-svc
+      image: broadinstitute/cromwell:84-3a19be6-SNAP
```

Comment:
how does this get automatically updated?

Reply:
Are you sure it is updated automatically? `3a19be6` is [two commits behind](https://github.com/broadinstitute/cromwell/commits/develop) the latest at `79afa29`.

We do see `79afa29` [here](https://github.com/broadinstitute/cromwhelm/pull/68/files#diff-e2f694d4842de8190150ccfc1ad19f5c04ba9d06cf0ffd3e46b0ef20e8ae917cR19-R20) where its update mechanism is the linked PR.

Reply:
nice catch!

Reply:
(I'm not sure that it's updated automatically; I must be wrong about that)